lib/robots/dor_repo/gis_delivery/load_vector.rb

```diff
@@ -60,6 +56,17 @@ def normalizer
         def rootdir
           @rootdir ||= GisRobotSuite.locate_druid_path bare_druid, type: :stage
         end
+
+        def run_shp2pgsql(projection, encoding, shp_filename, schema, sql_filename, stderr_filename)
+          # TODO: Perhaps put the .sql data into the content directory as .zip for derivative
+          # NOTE: -G for the geography column causes some issues with GeoServer
+          cmd = "shp2pgsql -s #{projection} -d -D -I -W #{encoding} " \
+                "'#{shp_filename}' #{schema}.#{bare_druid} " \
+                "> '#{sql_filename}' 2> '#{stderr_filename}'"
+          logger.debug "Running: #{cmd}"
+          system(cmd, exception: true)
+          raise "normalize-vector: #{bare_druid} shp2pgsql generated no SQL?" unless File.size?(sql_filename)
+        end
```

Comment:
as a bonus of moving this method into the one class where it's used, moving it here made it more sensible to test the actual `shp2pgsql` call along with the code that relies on it, instead of testing this method in isolation in the `VectorNormalizer` class (this method wasn't being tested at all before this PR).